From 8fcb61aace5296b9a3cb539fb69fe689769921d0 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Fri, 25 Sep 2015 10:20:32 +0800
Subject: [PATCH 0515/5242] MLK-11629 ARM: dts: imx7d-sdb: add sai<->sii902x
 sound card support

commit  ea531d9c3cf9721a514de7f20300351ad13717b0 from
https://source.codeaurora.org/external/imx/linux-imx.git

add sai<->sii902x sound card support for imx7d-sdb board

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 27298c4..d1e81660 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -180,6 +180,14 @@
 		assigned-clock-parents = <&clks IMX7D_PLL_AUDIO_POST_DIV>;
 		assigned-clock-rates = <0>, <12288000>;
 	};
+
+	sound-hdmi {
+		compatible = "fsl,imx7d-sdb-sii902x",
+			   "fsl,imx-audio-sii902x";
+		model = "sii902x-audio";
+		cpu-dai = <&sai1>;
+		hdmi-controler = <&sii902x>;
+	};
 };
 
 &adc1 {
-- 
1.7.9.5

